docs/pages/reference/config.mdx

```diff
@@ -786,6 +786,13 @@ proxy_service:
     #  enabled: yes
     #  email: user@example.com
 
+    # Configuration options for the Web UI served by the proxy service.
+    ui_config:
+      # The amount of scrollback in the terminal. Scrollback is the amount of
+      # rows that are retained when lines are scrolled beyond the initial
+      # viewport. Does not apply to session recording view.
+      scrollback_length: 1000
```

Comment:
1500 is the default now, right? Should we stick 1500 in the docs then?

Reply:
1000 is the default currently